src/transaction/client.rs

```diff
@@ -242,7 +242,7 @@ impl Client {
             batch_size: SCAN_LOCK_BATCH_SIZE,
             ..Default::default()
         };
-        self.cleanup_locks(&safepoint, options).await?;
+        self.cleanup_locks(&safepoint, vec![].., options).await?;
```

Comment:
`..` should be enough.

```suggestion
        self.cleanup_locks(&safepoint, .., options).await?;
```